Python/ast.c

```diff
@@ -3669,7 +3799,11 @@ ast_for_suite(struct compiling *c, const node *n)
         }
     }
     else {
-        for (i = 2; i < (NCH(n) - 1); i++) {
+        i = 2;
+        if (TYPE(CHILD(n, 1)) == TYPE_COMMENT)
+            i += 2;
```

Comment:
Maybe just in case add `REQ(CHILD(n, 2), NEWLINE)` just to be sure we don't eat anything meaningful.

Reply:
OK.